packages/twenty-front/src/modules/settings/components/SettingsNavigationDrawerItems.tsx

```diff
@@ -33,7 +33,9 @@ export const SettingsNavigationDrawerItems = () => {
   const isFunctionSettingsEnabled = useIsFeatureEnabled(
     'IS_FUNCTION_SETTINGS_ENABLED',
   );
+  const isFreeAccessEnabled = useIsFeatureEnabled('IS_FREE_ACCESS_ENABLED');
   const isCRMMigrationEnabled = useIsFeatureEnabled('IS_CRM_MIGRATION_ENABLED');
+  const isBillingEnabled = billing?.isBillingEnabled && !isFreeAccessEnabled;
```

Comment:
I would call this variable isBillingPageEnabled so it's not confusing with billing.isBillingEnabled